crates/tuono_lib/tests/utils/mod.rs

```diff
@@ -1,3 +1,108 @@
+use fs_extra::dir::create_all;
```

Comment:
`mod.rs` file should be just a tunnel for the files. Could you please put the implementation in a separate file?